packages/maskbook/src/plugins/Trader/UI/trending/TrendingCard.tsx

```diff
@@ -20,7 +20,9 @@ const useStyles = makeStyles((theme) => {
                               : 'rgba(101, 119, 134, 0.2) 0px 0px 15px, rgba(101, 119, 134, 0.15) 0px 0px 3px 1px'
                       }`,
                   }
-                : null),
```

Comment:
I could add this border conditionally to Minds only, but I figured maybe it will be pleasing to the eye also on other social media to have a very light border around the dialog